code/modules/ruins/lavaland_ruin_code.dm

```diff
@@ -1,31 +1,51 @@
+//If you're looking for spawners like ash walker eggs, check ghost_role_spawners.dm
+
 /obj/structure/lavaland_door
 	name = "necropolis gate"
-	desc = "An imposing, seemingly impenetrable door."
+	desc = "A tremendous and impossibly large gateway, bored into dense bedrock."
 	icon = 'icons/effects/96x96.dmi'
 	icon_state = "door"
 	anchored = 1
 	density = 1
+	opacity = 1
 	bound_width = 96
 	bound_height = 96
+	pixel_x = -32
+	bound_x = -32
 	burn_state = LAVA_PROOF
 	luminosity = 1
 	var/boss = FALSE
+	var/is_anyone_home = FALSE
```

Comment:
This is used for the "knocking on the door" fluff to prevent people from spamming the knock over and over.